.env

```
@@ -1,2 +1,2 @@
-POKEMON_API_BASE_URL =
+POKEMON_API_BASE_URL = "https://pokeapi.co/api/v2/"
```

small change, but convention says that last slash should be later added, url should not end with `/`, especially base url :)
```suggestion
POKEMON_API_BASE_URL = "https://pokeapi.co/api/v2"
```